lldb/source/Utility/AddressableBits.cpp

```diff
@@ -33,18 +33,26 @@ void AddressableBits::SetHighmemAddressableBits(
   m_high_memory_addr_bits = highmem_addressing_bits;
 }
 
+addr_t AddressableBits::AddressableBitToMask(uint32_t addressable_bits) {
+  assert(addressable_bits <= sizeof(addr_t) * 8);
```

Comment:
If we're in release mode, this assert isn't here. Is it possible for the user to set addresable_bits to > 64 via `settings set`?

If they do it's a bit of a garbage in garbage out situation, but wondering if we could add a check at that level too, or already have.

Reply:
I forgot that I was duplicating the addressable-bits-to-mask calculation in Process.cpp when I read the settings values.  I pushed a change to use the new static method in AddressableBits which includes the assert.